components/script/dom/xrview.rs

```diff
@@ -90,14 +88,18 @@ impl XRViewMethods for XRView {
     }
 
     /// https://immersive-web.github.io/webxr/#dom-xrview-projectionmatrix
-    fn ProjectionMatrix(&self, _cx: JSContext) -> NonNull<JSObject> {
-        if self.proj.get().is_null() {
+    fn ProjectionMatrix(&self, _cx: JSContext) -> Float32Array {
+        if !self.proj.is_initialized() {
             let cx = GlobalScope::get_cx();
             // row_major since euclid uses row vectors
             let proj = self.view.projection.to_array();
-            create_typed_array(cx, &proj, &self.proj);
+            let _ = self.proj.set_data(cx, &proj);
```

Comment:
set_data returns `Result<(), ()>` we should handle err:

```suggestion
            if self.proj.set_data(cx, &proj).is_err() {
                    // error handling
             }
```

Reply:
Yes, we should handle this.